docs/GettingStarted/configuration/ExporterFailure.md

```diff
@@ -407,3 +427,27 @@ Failure Time Exporter(s) configured to work with PagerDuty can be easily adjuste
 * Monitor issues of only specific urgencies, using [PAGERDUTY_URGENCY](#pagerduty_urgency).
 
 * Monitor issues of only specific priorities, using [PAGERDUTY_PRIORITY](#pagerduty_priority).
+
+## Configuring Azure DevOps
+
+### Default workflow
+
+By default, Failure Time Exporter(s) configured to work with Azure DevOps will:
+
+* Monitor all work items in all projects that live in Azure DevOps URL passed through [SERVER](#server) (that the token passed through [TOKEN](#token) has access to).
+
+* Monitor only work items tagged with `app.kubernetes.io/name=app_name`, where **app_name** is the name of one of the applications being monitored.
```

Comment:
wrong, as proven by tests. Update with something like "only issues tagged if proper label will be related to proper applications"

Reply:
I don't understand this statement.

Reply:
discussed along with **unknown** name setting